tools/topology/sof-tgl-rt711-rt1308.m4

```diff
@@ -58,6 +58,16 @@ ifdef(`EXT_AMP',
 '
 )
 
+# Add Bluetooth Audio Offload pass-through for ADL
+ifelse(PLATFORM, `adl', `
+	define(`BT_PIPELINE_PB_ID', `12')
+	define(`BT_PIPELINE_CP_ID', `13')
+	define(`BT_DAI_LINK_ID', eval(HDMI_BE_ID_BASE + 4))
+	define(`BT_PCM_ID', `10')
```

Comment:
Aa, found the issue with DMIC. The PCM IDs will move place. Works with following:
```
-       define(`BT_PCM_ID', `10')
+       ifelse(CHANNELS,`0', `define(`BT_PCM_ID', `10')', `define(`BT_PCM_ID', `12')')
```

Reply:
Thanks @kv2019i for troubleshooting the issue, will post v2 to fix, have a great weekend!

Reply:
In this code, the BT_PCM_ID is changeable based on whether there is DMIC or not. This will need extra effort to assign the PCM in UCM. (Do we need to support it in UCM?)

@plbossart Do you think we should use a fixed PCMID or a changeable PCMID?

Reply:
I am debating whether to use fixed PCM_ID or not for V2, you made a pretty strong case for fixed value, thanks @libinyang